static/templates/settings/account-settings.handlebars

```diff
@@ -132,5 +132,48 @@
             </div>
         </div>
 
+        <hr class="settings_separator" />
+
+        <div class="form-horizontal" id="api_key_button_box">
+            <div class="input-group side-padded-container">
+                <p class="api-key-note">
+                    {{#tr this}}
+                    For most bots using the API, you'll want to give each bot
+                    its own name and API key using the above section.  But if you
+                    want to write a bot that can access your own private
+                    messages, you should use your personal API key.
+                    {{/tr}}
```

Comment:
@vabs22 this text is totally wrong for the next context, since bots are no longer above.

@gnprice  can you propose some clearer text for this?  I think we can just say something like 
"We recommend using [bots](link) for most use of the Zulip API, but your personal API key is available where a bot is not appropriate."